### ISSUE TITLE
New squid breaks things

### ISSUE THREAD

**Issue by bhechinger** [thias/puppet-squid3#15]
In their infinite wisdom the squid folks have 'built in' the ACLs manager, localhost and to_localhost.

This means that putting those in the config file now causes this error:

```
Error: Could not start Service[squid3_service]: Execution of '/usr/sbin/squid3 start' returned 1: 2014/11/07 10:31:12| aclParseAclLine: ACL 'manager' already exists with different type.
FATAL: Bungled /etc/squid3/squid.conf line 642: acl manager proto cache_object
Squid Cache (Version 3.3.8): Terminated abnormally.
```

Squid Cache: Version 3.3.8

**Comment by victorgp** [comment]
Well, at this moment this module is completely unusable. Is there any chance this will be eventually fixed?

**Comment by webberist** [comment]
Since it's been 6 months since the squid update that breaks the config was
discovered I would support dropping support for the old squid going
forward. Lets just modify the config template fixing to work and be done
with it.

The dynamically determination of version to be installed and selecting
configs to match is difficult and never eventuated.

**Comment by thias** [comment]
This should have been fixed by the change in #13.
